tests/admin_tests/test_auth.py

```diff
@@ -137,7 +137,7 @@ def test_delete_instances(
 
 class TestAdminActionsAuthorization:
     actions = (
-        "reingest",
```

Comment:
Correct and true - but since we deploy PRs not commits I am ok enough with this.

But it's fair to call out because I'm sure I've said it's nice for all commits to be working things, so maybe I will fix this tomorrow 👀